docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -0,0 +1,307 @@
+   * removes the composite material variable (X_C) and the associated disintegration reaction
+   * adds 7 additional reactions
+
+This modified ADM1 property/reaction package:
+   * supports 'H2O', 'S_su', 'S_aa', 'S_fa', 'S_va', 'S_bu', 'S_pro', 'S_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_IP', 'S_I', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'X_PHA', 'X_PP', 'X_PAO', 'S_K', 'S_Mg', 'S_cat', 'S_an', and 'S_co2' as components
+   * supports only liquid and vapor phase
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'S_su', 'S_aa', 'S_fa', 'S_va', 'S_bu', 'S_pro', 'S_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_IP', 'S_I', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'X_PHA', 'X_PP', 'X_PAO', 'S_K', 'S_Mg', 'S_cat', 'S_an', 'S_co2']"
+  "Phases", ":math:`p`", "['Liq', 'Vap']"
+
+Components
+----------
+.. csv-table::
+  :header: "Description", "Symbol", "Variable"
+
+  "Monosaccharides, S_su", ":math:`S_{su}`", "S_su"
+  "Amino acids, S_aa", ":math:`S_{aa}`", "S_aa"
+  "Long chain fatty acids, S_fa", ":math:`S_{fa}`", "S_fa"
+  "Total valerate, S_va", ":math:`S_{va}`", "S_va"
+  "Total butyrate, S_bu", ":math:`S_{bu}`", "S_bu"
+  "Total propionate, S_pro", ":math:`S_{pro}`", "S_pro"
+  "Total acetate, S_ac", ":math:`S_{ac}`", "S_ac"
+  "Hydrogen gas, S_h2", ":math:`S_{h2}`", "S_h2"
+  "Methane gas, S_ch4", ":math:`S_{ch4}`", "S_ch4"
+  "Inorganic carbon, S_IC", ":math:`S_{IC}`", "S_IC"
+  "Inorganic nitrogen, S_IN", ":math:`S_{IN}`", "S_IN"
+  "Inorganic phosphorus, S_IP", ":math:`S_{IP}`", "S_IP"
+  "Soluble inerts, S_I", ":math:`S_I`", "S_I"
+  "Carbohydrates, X_ch", ":math:`X_{ch}`", "X_ch"
+  "Proteins, X_pr", ":math:`X_{pr}`", "X_pr"
+  "Lipids, X_li", ":math:`X_{li}`", "X_li"
+  "Sugar degraders, X_su", ":math:`X_{su}`", "X_su"
+  "Amino acid degraders, X_aa", ":math:`X_{aa}`", "X_aa"
+  "Long chain fatty acid (LCFA) degraders, X_fa", ":math:`X_{fa}`", "X_fa"
+  "Valerate and butyrate degraders, X_c4", ":math:`X_{c4}`", "X_c4"
+  "Propionate degraders, X_pro", ":math:`X_{pro}`", "X_pro"
+  "Acetate degraders, X_ac", ":math:`X_{ac}`", "X_ac"
+  "Hydrogen degraders, X_h2", ":math:`X_{h2}`", "X_h2"
+  "Particulate inerts, X_I", ":math:`X_I`", "X_I"
+  "Polyhydroxyalkanoates, X_PHA", ":math:`X_{PHA}`", "X_PHA"
+  "Polyphosphates, X_PP", ":math:`X_{PP}`", "X_PP"
+  "Phosphorus accumulating organisms, X_{PAO}", ":math:`X_PAO`", "X_PAO"
+  "Potassium, S_K", ":math:`S_K`", "S_K"
+  "Magnesium, S_Mg", ":math:`S_{Mg}`", "S_Mg"
+  "Total cation equivalents concentration, S_cat", ":math:`S_{cat}`", "S_cat"
+  "Total anion equivalents concentration, S_an", ":math:`S_{an}`", "S_an"
+  "Carbon dioxide carbon, S_co2", ":math:`S_{co2}`", "S_co2"
```

Comment:
Why not just "carbon dioxide"?

Reply:
And is dissolved CO2 (liquid phase) not considered in the model? I see the note below that it is only in the vapor phase, but I'd think that some dissolved CO2 would be involved. Just a sanity check.

Reply:
"Carbon dioxide carbon" was just copied over from the ADM1 documentation. I can update both of them unless there's some specific reason this was included @agarciadiego

Reply:
I don't think any dissolved CO2 is involved. For that to be the case, wouldn't CO2 need to be listed as a liquid phase component under `modified_adm1_properties.py`?

Reply:
Yes- I guess it's more of a question on how it was treated in the literature's representation of the model.

Reply:
@adam-a-a The anaerobic digestor unit model handles gas-liquid transfer here, so liquid CO2 is considered in the unit model, but Chenyu and I haven't changed anything related to vapor properties or gas-liquid transfer with the modified ADM1 property/reaction models.
![image](https://user-images.githubusercontent.com/96305519/229568657-bde6fc34-ebfb-458c-88c4-ddc4c11c3126.png)